test/fixtures/mocks/express_request.js

```diff
@@ -9,12 +10,15 @@ module.exports = function (body) {
     query: {
       title: 'foo-bar-baz',
       unSupportedKey: 'something',
-      include: 'books'
+      include: 'book,chapter'
     },
     body: {
       mock: {
-        key: 'value'
+        key: 'vaulue'
       }
+    },
+    headers: {
+      'content-type': 'application/vnd.api+json'
```

Comment:
This isn't necessary on read/destroy, and we shouldn't assume it's sent by default.